### PR TITLE
Add delete for departments and small fix for front

### PR DESCRIPTION
**Pull request by GrishaShats**

### CODE REVIEW

File: server/api/v1/departments.route.js [hunk 1]
```diff
@@ -28,17 +32,39 @@ router.route('/departments')
...
     });
+  })
+  .delete(async (req, res) => {
+    const { id } = req.body;
+    await Departments.findByIdAndDelete(id, (err) => {
+      if (err) {
+        res.status(500).json({
+          err,
+        });
+      } else if (!id) {
```
Comment (ilyachenko) [inline server/api/v1/departments.route.js]: do we need it?

Comment (ilyachenko) [reply server/api/v1/departments.route.js]: I mean `else if`

File: server/api/v1/departments.route.js [hunk 2]
```diff
@@ -5,17 +5,21 @@ const router = express.Router();
 
 router.route('/departments')
   .get(async (req, res) => {
-    await Departments.find().populate('employees', ['firstName', 'lastName'])
+    await Departments.find()
```
Comment (dd1mk) [inline server/api/v1/departments.route.js]: unnecessary async/await

File: server/api/v1/departments.route.js [hunk 3]
```diff
@@ -28,17 +32,39 @@ router.route('/departments')
...
+        });
+      }
+      res.json({
+        deleted: 'Successfully',
+      });
+    });
   });
 router.get('/departments/:id', async (req, res) => {
   const { id } = req.params;
   await Departments.findById(id, (err, department) => {
```
Comment (dd1mk) [inline server/api/v1/departments.route.js]: also unnecessary async/await

File: server/api/v1/issues.route.js [hunk 4]
```diff
@@ -79,9 +88,13 @@ router.route('/issues')
     const { id } = req.body;
     await Issues.findByIdAndDelete(id, (err) => {
```
Comment (dd1mk) [inline server/api/v1/issues.route.js]: and here)

**Pull request merged**